src/grid/grid.directive.ts

```diff
@@ -0,0 +1,86 @@
+import { Directive, HostBinding, Input, OnInit } from '@angular/core';
+
+@Directive({
+  selector: '[ibmGrid]',
+})
+export class GridDirective implements OnInit {
+  @Input()
+  class = '';
+
+  @HostBinding('class')
+  gridClasses = 'bx--grid';
```

Comment:
I think `@HostBinding("class.bx--grid") = true` might give you the functionality you expect here, without needing to manually build/check the class input.